controllers/feeds/getFeedChat.js

```diff
@@ -261,6 +265,8 @@ const getFeedChatService = async (req, res) => {
           isSeen: b.isSeen,
           totalComment: totalComment + commentLevel2 + total3,
           isOwnPost,
+          isOwnAnonymousPost,
+          isOwnSignedPost,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variables `isOwnAnonymousPost` and `isOwnSignedPost` are now included in the returned object. This seems to be consistent with the PR's purpose of distinguishing between anonymous and signed posts.
```diff
+ isOwnAnonymousPost,
+ isOwnSignedPost,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->